src/services/automation/editor/components/initialConditions/PushNotificationCondition.js

```diff
@@ -20,6 +20,9 @@
       PushNotificationCondition.prototype.checkCompleted = function() {
         if (this.sendType === SEND_TYPE.INMEDIATE) {
           this.completed = this.domains.length > 0;
+        } else if (this.sendType === SEND_TYPE.SCHEDULED_DATE) {
+          this.completed = this.frequency !== null && this.frequency.type !== 'date' 
```

Comment:
Este no debiera ser asi?
```suggestion
          this.completed = this.frequency !== null && this.frequency.type !== FREQUENCY_TYPE.DATE
```

Reply:
si, de cualquier manera estoy pensando si cada vez que elegimos un nuevo metodo de envio (Inmediato, programado o peridodico deberia hacer un reset del objeto frecuencia aunque asi perderiamos si seteamos una hora con su timezone, o hacer un reset al type